arbor/mc_cell_group.hpp

```diff
@@ -57,10 +57,26 @@ class mc_cell_group: public cell_group {
 
     void remove_all_samplers() override;
 
+    void generate_deps_gids(const recipe& rec, std::vector<cell_gid_type>);
+
+    std::vector<cell_gid_type> get_gids() {
+        return gids_;
+    }
+
+    std::vector<int> get_dependencies() {
+        return deps_;
+    }
+
 private:
     // List of the gids of the cells in the group.
     std::vector<cell_gid_type> gids_;
 
+    // Permutation of the gids of the cells in the group.
```

Comment:
Because it can be confusing, the comment should describe explicitly which way the permutation maps, i.e. `perm_gids_[i]` is the index of `gids_[i]` in the original `gids` vector passed to the ctor. (Perhaps it would be clearer if `gids_` were renamed `reordered_gids_`? That said, this will be different again when the integration domain change is made.)

Reply:
I added a clearer comment.